src/main/resources/com/uipath/uipathpackage/UiPathPack/help-orchestratorAddress.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Specify the address of the Orchestrator instance onto which the package(s) will be packed.
```

Comment:
Specify the address of the Orchestrator instance from which library dependencies should be restored.